libp2p/stream/chronosstream.nim

```diff
@@ -21,6 +21,9 @@ const
 type
   ChronosStream* = ref object of Connection
     client: StreamTransport
+    tracked: bool
+
+declarePublicCounter(libp2p_peers_identity, "peers identities", labels = ["agent"])
```

Comment:
We also want to track the total amount of bytes sent/rcvd, this is probably better as a gauge.